pom.xml

```diff
@@ -36,15 +36,15 @@ POSSIBILITY OF SUCH DAMAGE.
   <modelVersion>4.0.0</modelVersion>
 
   <groupId>gov.nasa.pds</groupId>
-  <artifactId>registry</artifactId>
+  <artifactId>registry-mgr-legacy</artifactId>
   <version>2.3.0-SNAPSHOT</version>
   <packaging>jar</packaging>
 
-  <name>Registry</name>
+  <name>Legacy Registry Manager Solr</name>
   <description>
-    The Registry provides provides the PDS-specific search protocol and the search capability for the PDS search index generated through the Search Core software. The core functionality for this service is satisfied by Apache Solr.
+    The Legacy Registry provides provides the PDS-specific search protocol and the search capability for the PDS search index generated through the Search Core software. The core functionality for this service is satisfied by Apache Solr.
   </description>
-  <url>https://nasa-pds-incubator.github.io/registry/</url>
+  <url>https://nasa-pds.github.io/registry-mgr-legacy/</url>
```

Comment:
(repeat) https://nasa-pds.github.io/registry-mgr-legacy/ is 404. Is this a TODO item?

Reply:
This should go live with release. If not, we can update this later. Doesn't really matter all too much since this is an internal service.